examples/anchors.yaml

```diff
@@ -32,7 +32,9 @@ meter_provider:
             <<: *otlp-exporter
             endpoint: http://localhost:4318/v1/metrics
             temporality_preference: delta
-            default_histogram_aggregation: base2_exponential_bucket_histogram
+            default_aggregation_by_instrument_kind:
```

Comment:
what if I want to
- have an aggregation preference for all kinds?
- have settings that apply for all exporters?

I see that there is a yaml anchor - but I'm not sure this should be the only option to share values

Reply:
Using anchors makes using the config easier. No need to merge properties from some global exporter config with per-exporter config. And simplifies the schema.

Reply:
it's a lesser known and complicated feature